*** tests/A-07-05-M-ok.out	2022-10-02 16:28:16.725473000 +0100
--- tests/A-07-05-M-ok.outhyp	2022-10-17 22:21:49.696305536 +0100
***************
*** 11,28 ****
  2 - Registar terminal
  3 - Operações de terminal
  0 - Sair
! Escolha uma opção: Número do terminal: Tipo de terminal (BASIC ou FANCY): Identificador do cliente: GESTÃO DE TERMINAIS
  1 - Mostrar todos os terminais
  2 - Registar terminal
  3 - Operações de terminal
  0 - Sair
! Escolha uma opção: Número do terminal: Tipo de terminal (BASIC ou FANCY): Identificador do cliente: Registar terminal: Operação inválida: O terminal '100009' já existe.
  GESTÃO DE TERMINAIS
  1 - Mostrar todos os terminais
  2 - Registar terminal
  3 - Operações de terminal
  0 - Sair
! Escolha uma opção: BASIC|100009|cli002|IDLE|0|0
  GESTÃO DE TERMINAIS
  1 - Mostrar todos os terminais
  2 - Registar terminal
--- 11,29 ----
  2 - Registar terminal
  3 - Operações de terminal
  0 - Sair
! Escolha uma opção: Insert terminal's ID:Insert Terminal's Type (BASIC or FANCY):Insert client:Terminal registered
! GESTÃO DE TERMINAIS
  1 - Mostrar todos os terminais
  2 - Registar terminal
  3 - Operações de terminal
  0 - Sair
! Escolha uma opção: Insert terminal's ID:Insert Terminal's Type (BASIC or FANCY):Insert client:Registar terminal: Operação inválida: O terminal '100009' já existe.
  GESTÃO DE TERMINAIS
  1 - Mostrar todos os terminais
  2 - Registar terminal
  3 - Operações de terminal
  0 - Sair
! Escolha uma opção: BASIC|100009|cli002|IDLE|0.0|0.0
  GESTÃO DE TERMINAIS
  1 - Mostrar todos os terminais
  2 - Registar terminal
