*** tests/A-08-06-M-ok.out	2022-10-05 21:24:49.207103000 +0100
--- tests/A-08-06-M-ok.outhyp	2022-10-17 23:07:33.726656915 +0100
***************
*** 11,25 ****
  2 - Registar terminal
  3 - Operações de terminal
  0 - Sair
! Escolha uma opção: Número do terminal: Tipo de terminal (BASIC ou FANCY): Identificador do cliente: GESTÃO DE TERMINAIS
  1 - Mostrar todos os terminais
  2 - Registar terminal
  3 - Operações de terminal
  0 - Sair
! Escolha uma opção: BASIC|969001|cli001|IDLE|0|0
! BASIC|969002|cli002|IDLE|0|0
! BASIC|969003|cli003|IDLE|0|0
! BASIC|999000|cli001|IDLE|0|0
  GESTÃO DE TERMINAIS
  1 - Mostrar todos os terminais
  2 - Registar terminal
--- 11,26 ----
  2 - Registar terminal
  3 - Operações de terminal
  0 - Sair
! Escolha uma opção: Número do terminal: Insert Terminal's Type (BASIC or FANCY):Identificador do cliente: Terminal registered
! GESTÃO DE TERMINAIS
  1 - Mostrar todos os terminais
  2 - Registar terminal
  3 - Operações de terminal
  0 - Sair
! Escolha uma opção: BASIC|969003|cli003|IDLE|0.0|0.0
! BASIC|969002|cli002|IDLE|0.0|0.0
! BASIC|969001|cli001|IDLE|0.0|0.0
! BASIC|999000|cli001|IDLE|0.0|0.0
  GESTÃO DE TERMINAIS
  1 - Mostrar todos os terminais
  2 - Registar terminal
