*** tests/A-08-04-M-ok.out	2022-10-17 22:32:26.051738121 +0100
--- tests/A-08-04-M-ok.outhyp	2022-10-18 00:28:10.327698764 +0100
***************
*** 11,19 ****
  2 - Registar terminal
  3 - Operações de terminal
  0 - Sair
! Escolha uma opção: BASIC|969001|cli001|IDLE|0|0
  BASIC|969002|cli002|IDLE|0|0
! BASIC|969003|cli003|IDLE|0|0
  GESTÃO DE TERMINAIS
  1 - Mostrar todos os terminais
  2 - Registar terminal
--- 11,19 ----
  2 - Registar terminal
  3 - Operações de terminal
  0 - Sair
! Escolha uma opção: BASIC|969003|cli003|IDLE|0|0
  BASIC|969002|cli002|IDLE|0|0
! BASIC|969001|cli001|IDLE|0|0
  GESTÃO DE TERMINAIS
  1 - Mostrar todos os terminais
  2 - Registar terminal
