*** tests/A-09-02-M-ok.out	2022-10-10 10:30:18.141778000 +0100
--- tests/A-09-02-M-ok.outhyp	2022-10-17 22:21:50.212275106 +0100
***************
*** 11,17 ****
  2 - Registar terminal
  3 - Operações de terminal
  0 - Sair
! Escolha uma opção: Número do terminal: OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
  3 - Desligar
--- 11,17 ----
  2 - Registar terminal
  3 - Operações de terminal
  0 - Sair
! Escolha uma opção: Insert terminal id:OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
  3 - Desligar
***************
*** 21,27 ****
  7 - Consultar saldo
  8 - Enviar comunicação de texto
  9 - Iniciar comunicação
! 11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: GESTÃO DE TERMINAIS
  1 - Mostrar todos os terminais
--- 21,27 ----
  7 - Consultar saldo
  8 - Enviar comunicação de texto
  9 - Iniciar comunicação
! 10 - Terminar comunicação activa
  0 - Sair
  Escolha uma opção: GESTÃO DE TERMINAIS
  1 - Mostrar todos os terminais
