*** tests/A-09-03-M-ok.out	2022-10-17 22:32:26.052738133 +0100
--- tests/A-09-03-M-ok.outhyp	2022-10-18 00:28:10.641701100 +0100
***************
*** 21,26 ****
--- 21,27 ----
  7 - Consultar saldo
  8 - Enviar comunicação de texto
  9 - Iniciar comunicação
+ 10 - Terminar comunicação activa
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: GESTÃO DE TERMINAIS
***************
*** 38,43 ****
--- 39,45 ----
  7 - Consultar saldo
  8 - Enviar comunicação de texto
  9 - Iniciar comunicação
+ 10 - Terminar comunicação activa
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: GESTÃO DE TERMINAIS
***************
*** 55,60 ****
--- 57,63 ----
  7 - Consultar saldo
  8 - Enviar comunicação de texto
  9 - Iniciar comunicação
+ 10 - Terminar comunicação activa
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: GESTÃO DE TERMINAIS
