*** tests/A-09-01-M-ok.out	2022-10-10 09:58:27.357984000 +0100
--- tests/A-09-01-M-ok.outhyp	2022-10-17 22:21:50.136279581 +0100
***************
*** 11,17 ****
  2 - Registar terminal
  3 - Operações de terminal
  0 - Sair
! Escolha uma opção: Número do terminal: Operações de terminal: Operação inválida: O terminal '969000' não existe.
  GESTÃO DE TERMINAIS
  1 - Mostrar todos os terminais
  2 - Registar terminal
--- 11,17 ----
  2 - Registar terminal
  3 - Operações de terminal
  0 - Sair
! Escolha uma opção: Insert terminal id:Operações de terminal: Operação inválida: O terminal '969000' não existe.
  GESTÃO DE TERMINAIS
  1 - Mostrar todos os terminais
  2 - Registar terminal
