*** testsSol/A-15-07-M-ok.out	2022-11-04 13:24:35.611974000 +0000
--- testsSol/A-15-07-M-ok.outhyp	2022-11-04 14:16:42.381041971 +0000
***************
*** 229,235 ****
  0 - Sair
  Escolha uma opção: 10
  Duração da comunicação (em minutos): 1
! Custo da comunicação: 20 unidades.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 229,235 ----
  0 - Sair
  Escolha uma opção: 10
  Duração da comunicação (em minutos): 1
! Custo da comunicação: 10 unidades.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
***************
*** 267,273 ****
  6 - Mostrar saldo de cliente
  0 - Sair
  Escolha uma opção: 2
! CLIENT|cli001|Manuel Pinheiro|103443|PLATINUM|YES|4|900|90
  CLIENT|cli002|Pedro Pinheiro|103447|NORMAL|YES|2|0|0
  CLIENT|cli003|Pedro Oliveira|103449|NORMAL|YES|3|0|0
  CLIENT|cli201|Ludgero Oliveira|103440|NORMAL|YES|0|0|0
--- 267,273 ----
  6 - Mostrar saldo de cliente
  0 - Sair
  Escolha uma opção: 2
! CLIENT|cli001|Manuel Pinheiro|103443|GOLD|YES|4|900|80
  CLIENT|cli002|Pedro Pinheiro|103447|NORMAL|YES|2|0|0
  CLIENT|cli003|Pedro Oliveira|103449|NORMAL|YES|3|0|0
  CLIENT|cli201|Ludgero Oliveira|103440|NORMAL|YES|0|0|0
