*** testsBoth/A-12-05-M-ok.out	2022-10-30 22:57:52.837158000 +0000
--- testsBoth/A-12-05-M-ok.outhyp	2022-11-01 17:27:28.362895972 +0000
***************
*** 27,35 ****
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: 9
- Número do terminal: 969008
- Tipo de comunicação a iniciar (VOICE, VIDEO): VOICE
- O terminal de destino ('969008') está em silêncio.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 27,32 ----
***************
*** 42,49 ****
  9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: 11
- Não existe nenhuma comunicação em curso.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 39,61 ----
  9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
+ Escolha uma opção: 969008
+ Opção inválida!
+ OPERAÇÕES DO TERMINAL
+ 1 - Ligar
+ 2 - Colocar no silêncio
+ 3 - Desligar
+ 4 - Adicionar amigo
+ 5 - Retirar amigo
+ 6 - Pagamento
+ 7 - Consultar saldo
+ 8 - Enviar comunicação de texto
+ 9 - Iniciar comunicação
+ 11 - Mostrar comunicação em curso
+ 0 - Sair
+ Escolha uma opção: VOICE
+ Número inválido!
  Escolha uma opção: 11
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
