*** testsBoth/A-15-05-M-ok.out	2022-10-30 22:57:55.127603000 +0000
--- testsBoth/A-15-05-M-ok.outhyp	2022-11-01 18:10:39.834474225 +0000
***************
*** 27,34 ****
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: 9
- Número do terminal: 969014
- Tipo de comunicação a iniciar (VOICE, VIDEO): VIDEO
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 27,32 ----
***************
*** 37,48 ****
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
! 10 - Terminar comunicação activa
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: 10
! Duração da comunicação (em minutos): 30
! Custo da comunicação: 900 unidades.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 35,76 ----
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
! 8 - Enviar comunicação de texto
! 9 - Iniciar comunicação
! 11 - Mostrar comunicação em curso
! 0 - Sair
! Escolha uma opção: 969014
! Opção inválida!
! OPERAÇÕES DO TERMINAL
! 1 - Ligar
! 2 - Colocar no silêncio
! 3 - Desligar
! 4 - Adicionar amigo
! 5 - Retirar amigo
! 6 - Pagamento
! 7 - Consultar saldo
! 8 - Enviar comunicação de texto
! 9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
+ Escolha uma opção: VIDEO
+ Número inválido!
  Escolha uma opção: 10
! Opção inválida!
! OPERAÇÕES DO TERMINAL
! 1 - Ligar
! 2 - Colocar no silêncio
! 3 - Desligar
! 4 - Adicionar amigo
! 5 - Retirar amigo
! 6 - Pagamento
! 7 - Consultar saldo
! 8 - Enviar comunicação de texto
! 9 - Iniciar comunicação
! 11 - Mostrar comunicação em curso
! 0 - Sair
! Escolha uma opção: 30
! Opção inválida!
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
***************
*** 81,87 ****
  7 - Mostrar terminais com saldo positivo
  0 - Sair
  Escolha uma opção: 1
- VIDEO|1|969011|969014|30|900|FINISHED
  CONSULTAS
  1 - Mostrar todas as comunicações
  2 - Mostrar comunicações feitas por um cliente
--- 109,114 ----
